index.js

```diff
@@ -62,24 +62,16 @@ module.exports = function (options) {
 			if (errorList.length > 0) {
 				file.jscs.success = false;
 				file.jscs.errorCount = errorList.length;
-				file.jscs.errors = errorList;
+				file.jscs.errors = errors;
```

Comment:
Because JSCS reporters expect an array of errorCollections, not errorLists. Passing an errorList would throw an error, since the reporters call `.getErrorList()` on the [error collections passed as argument](https://github.com/jscs-dev/node-jscs/blob/394edc90eb6b012852c55ddd0868656455b8fec0/lib/reporters/console.js#L14).